packages/lodestar/src/api/rest/routes/validator/produceAttestation.ts

```diff
@@ -0,0 +1,60 @@
+import {IFastifyServer} from "../../index";
+import fastify, {DefaultQuery} from "fastify";
+import {IApiModules} from "../../../interface";
+import {IncomingMessage, Server, ServerResponse} from "http";
+import {toRestJson} from "../../utils";
+import {produceAttestation} from "../../../impl/validator";
+
+interface Query extends DefaultQuery {
+  validator_pubkey: string;
+  poc_bit: number;
+  slot: number;
+  shard: number;
+}
+
+
+const opts: fastify.RouteShorthandOptions<Server, IncomingMessage, ServerResponse, Query> = {
+  schema: {
+    querystring: {
+      type: 'object',
+      required: ["validator_pubkey", "poc_bit", "slot", "shard"],
+      properties: {
+        "randvalidator_pubkeyao_reveal": {
+          type: "string"
+        },
+        "poc_bit": {
+          type: "integer",
+          minimum: 0
+        },
+        shard: {
+          type: "integer",
+          minimum: 0
+        },
+        slot: {
+          type: "integer",
+          minimum: 0
+        },
+      }
+    },
+  }
+};
+
+export const registerAttestationProductionEndpoint = (fastify: IFastifyServer, modules: IApiModules): void => {
+  fastify.get<Query>(
+    '/attestation',
+    opts,
+    async (request, reply) => {
+      const attestation = await produceAttestation(
+        modules.config,
+        modules.db,
+        modules.chain,
+        request.query.shard,
+        request.query.slot
```

Comment:
How about `validator_pubkey` and `poc_bit` query param as in https://github.com/ethereum/eth2.0-APIs/blob/9daa854389c774335b4c39350b4b2f5faa0aa409/apis/validator/beacon-node-oapi.yaml#L194?

Reply:
Yeah, need to checkout where does it fits :sweat_smile: